src/v/raft/tests/raft_group_fixture.h

```diff
@@ -667,6 +667,12 @@ ss::future<bool> retry_with_leader(
                    .handle_exception([meta](const std::exception_ptr&) {
                        meta->success = false;
                        meta->current_retry++;
+                   })
+                   .then([meta] {
+                       if (!meta->success) {
+                           return ss::sleep(200ms * meta->current_retry);
```

Comment:
10?ms ? also sleep_abortable?

Reply:
this is for testing purpose only, doesn't have to do abortable, also `200ms` is more realistic value for CI